website_seo_sale/__openerp__.py

```diff
@@ -0,0 +1,22 @@
+{
+    'name': 'eCommerce SEO Optimized',
+    'category': 'Website',
+    'sequence': 55,
+    'summary': 'Sell Your Products Online',
+    'website': 'http://www.bloopark.de',
+    'version': '1.0',
+    'description': """
+OpenERP E-Commerce SEO Optimized
+==================
+
+        """,
+    'depends': [ 'website_sale',],
```

Comment:
I think we should use the module 'website_seo' from suite as dependency. It has some general features about seo that maybe can be useful in this context. This module should be the core of seo suite. Is possible we need to do some improvements in order to be useful for website_seo_sale. Could you take a look on it? What do you think?